spark-connector/src/main/java/com/datastrato/gravitino/spark/connector/table/SparkBaseTable.java

```diff
@@ -117,6 +121,15 @@ public WriteBuilder newWriteBuilder(LogicalWriteInfo info) {
     return ((SupportsWrite) getSparkTable()).newWriteBuilder(info);
   }
 
+  @Override
+  public Transform[] partitioning() {
```

Comment:
The advantage of this approach is that we don't need to check whether the 'Transform' between gravitino and datasource is correct, because datasource already guarantees its correctness

Reply:
because we want get all schema info from Gravitino

Reply:
Hi @FANNG1 could you give an example to show the difference between them?

Reply:
There is no difference,  I think this's the same question why not get table schema from lazysparktable?  By design we want to get most information from gravitino as much as possible.